contrib/mixin/mixin.libsonnet

```diff
@@ -130,7 +130,7 @@
               severity: 'critical',
             },
             annotations: {
-              description: 'etcd cluster "{{ $labels.job }}": gRPC requests to {{ $labels.grpc_method }} are taking {{ $value }}s on etcd instance {{ $labels.instance }}.',
+              description: 'etcd cluster "{{ $labels.job }}": 99 % of gRPC requests to {{ $labels.grpc_method }} are taking {{ $value }}s on etcd instance {{ $labels.instance }}.',
```

Comment:
optional nit: we use 99 % here and 99th percentile below we could align.

Reply:
Done, good catch, I also rephrased it a bit.